pkg/storage/system_intake.go

```diff
@@ -246,6 +248,37 @@ func (s *Store) FetchSystemIntakeByID(ctx context.Context, id uuid.UUID) (*model
 		}
 	}
 
+	sources := []*models.SystemIntakeFundingSource{}
+
+	// TODO: Rather than two separate queries/round-trips to the database, the funding sources should be
```

Comment:
Since we're using two separate queries, should we wrap both the intake fetching and funding source fetching in a transaction?

Reply:
Since this ticket's been blocking Ashley for a long time now, I feel like this is something that could be another small tech debt ticket?

Reply:
I figured that it'd be pretty straightforward to implement, just prefix the SELECTs with
```go
tx := s.db.MustBegin()
defer tx.Rollback()
```
then run the queries off of `tx` instead of `s.db`. 

That said, this wouldn't help with the scenario I was thinking of (where an intake and/or associated funding sources get modified between fetching the intake and fetching funding sources). Reading the [Postgres docs around transaction isolation](https://www.postgresql.org/docs/11/transaction-iso.html), I think that Postgres's default isolation level of "read committed" wouldn't prevent this, even if both queries here were wrapped in a transaction; it specifically says "Also note that two successive SELECT commands can see different data, even though they are within a single transaction, if other transactions commit changes after the first SELECT starts and before the second SELECT starts." While we could theoretically change the Postgres configuration, that's definitely outside the scope of this work, and it'd be easier just to figure out how to write this as a single query.

tl;dr: Grouping the two queries in a transaction wouldn't help anyways, we can look into rewriting them as a single query as a tech debt item.